libs/designsystem/src/lib/components/button/button.component.integration.spec.ts

```diff
@@ -385,7 +385,7 @@ describe('ButtonComponent configured with icon only', () => {
     });
 
     describe('and configured with attentionlevel', () => {
-      attentionLevels.forEach((attentionLevel: AttentionLevel) => {
+      attentionLevels.forEach((attentionLevel) => {
```

Comment:
Consider adding a similar test like these to the undecorated button.

Reply:
-considered and done...